### ISSUE TITLE
waspmote

### ISSUE THREAD

**Issue by jdavid** [spectraphilic/wsn_riot#2]
List of things we need before we can use RIOT with the waspmote-pro board (non exhaustive):

- [x] Support for timers, follow https://github.com/RIOT-OS/RIOT/issues/14750
- [ ] Support power saving
- [ ] RTC DS1337C support (v15), issue #3 
- [ ] RTC DS3231SN support (v12) *
- [ ] periph_rtc feature support in waspmote-pro board
- [ ] Use program memory, our code has to use progmem as much as possible (and still work with non avr boards)

(*) We don't use v12 but the waspmote-pro board in RIOT was developed for v12, so to have periph_rtc support in waspmote-pro we will likely need to get DS3231SN working.

**Comment by jdavid** [comment]
Closing in favor of individual issues.
